e2e/optimism_test.go

```diff
@@ -131,11 +131,11 @@ func (a *L2AltDA) ActL1Finalized(t actions.Testing) {
 }
 
 func TestOptimismKeccak256Commitment(gt *testing.T) {
```

Comment:
```suggestion
func TestOptimismKeccak256Commitment(t *testing.T) {
```

Reply:
Fixed in [this huge commit](https://github.com/Layr-Labs/eigenda-proxy/pull/337/commits/78fc10bdfc142b46489506ee39e13af3e94d0c20)